src/app/helpers/searchFunctions.js

```diff
@@ -12,6 +12,11 @@ const SearchFunctions = {
   [FilterModes.CONTAINS]: f.curry((stringOfFilters, str) => {
     return f.every(f.contains(f, clean(str)), f.words(clean(stringOfFilters)));
   }),
+  [FilterModes.STATUS]: f.curry((stringOfFilters, shouldContain, str) => {
```

Comment:
Das bricht jetzt natürlich den Kontrakt der Filterfunktionen, weil der Type anders ist. In allen anderen Fällen ist es `Query -> Element -> Boolean` und hier ist es `Query -> Boolean -> Element -> Boolean` -- die Funktionen können nicht mehr im selben Kontext verwendet werden, was der eigentliche Grund für diese Map ist.

Da sich die Suche nach den Tags so fundamental unterscheidet, sollte das vermutlich gar nicht in dieser Map für Stringvergleiche drinnen sein. Man kanns ja in der selben Datei lassen und einzeln exportieren -- evtl. kommen da ja noch mehrere Tag-Wert-Suchen dazu...